elastalert/elastalert.py

```diff
@@ -657,6 +669,12 @@ def run_query(self, rule, start=None, end=None, scroll=False):
                 rule_inst.add_aggregation_data(data)
             else:
                 rule_inst.add_data(data)
+        if isinstance(rule['type'], FlatlineRule):
+            if "writeback_events_up_enabled" in rule_inst.rules and rule_inst.rules["writeback_events_up_enabled"] and len(rule_inst.events_up) !=0:
```

Comment:
Can you replace `"writeback_events_up_enabled" in rule_inst.rules and rule_inst.rules["writeback_events_up_enabled"]` with `rule_inst.get("writeback_events_up_enabled")` ?

Also add a space between != and 0